commit 3a5699d58735717ac70b443581e18c2a60f162c5
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Thu Mar 1 21:36:30 2018 +0000

    overlays: Add updated mmc1 alias to sdio overlays
    
    In the downstream RPi kernel, aliases are used to assign instance
    numbers to the SD/MMC interfaces. The updated sdio overlays
    effectively rename the device node, so the mmc1 alias has to be
    updated in order to preserve the mmc1 numbering, otherwise the device
    will appear as mmc2.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/overlays/sdio-1bit-overlay.dts b/arch/arm/boot/dts/overlays/sdio-1bit-overlay.dts
index c0898dc57036..297daae8faf9 100644
--- a/arch/arm/boot/dts/overlays/sdio-1bit-overlay.dts
+++ b/arch/arm/boot/dts/overlays/sdio-1bit-overlay.dts
@@ -48,6 +48,14 @@
 		};
 	};
 
+	fragment@3 {
+		target-path = "/aliases";
+		__overlay__ {
+			mmc1 = "/soc/sdio@7e300000";
+		};
+	};
+
+
 	__overrides__ {
 		poll_once = <&sdio_1bit>,"non-removable?";
 		sdio_overclock = <&sdio_1bit>,"brcm,overclock-50:0";
diff --git a/arch/arm/boot/dts/overlays/sdio-overlay.dts b/arch/arm/boot/dts/overlays/sdio-overlay.dts
index 2ac7d4f5f81d..685b65a858e8 100644
--- a/arch/arm/boot/dts/overlays/sdio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/sdio-overlay.dts
@@ -48,6 +48,13 @@
 		};
 	};
 
+	fragment@3 {
+		target-path = "/aliases";
+		__overlay__ {
+			mmc1 = "/soc/sdio@7e300000";
+		};
+	};
+
 	__overrides__ {
 		poll_once = <&sdio_ovl>,"non-removable?";
 		bus_width = <&sdio_ovl>,"bus-width:0";
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1198_3a5699d58735717ac70b443581e18c2a60f162c5.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1198_3a5699d58735717ac70b443581e18c2a60f162c5.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
